portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,37 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> quotes;
+
+  @Override
+  public void init(){
+    quotes = new ArrayList<>();
+    quotes.add("You must be the change you wish to see in the world. - Gandhi");
+    quotes.add("Creativity is intelligence having fun. - Albert Einstein");
+    quotes.add("Whether you think you can or you think you can't, you're right. - Henry Ford");
+    quotes.add("Go confidently in the direction of your dreams.  Live the life you have imagined. - Henry David Thoreau");
```

Comment:
general rule of thumb is that code should not go past 100 characters (including the white space) aka shouldn't go past the 100 column.